services/theme_update.py

```diff
@@ -222,7 +222,7 @@ def application(environ, start_response):
                 return ['']
             start_response(status, update.get_headers(len(output)))
         except:
-            log_exception(data)
```

Comment:
`data` still looks important. Maybe turn this into `log.error('{}; data={}'.format(exc, data))` ?